draft-irtf-cfrg-hpke.md

```diff
@@ -1166,6 +1171,10 @@ of the label used as parameter to `LabeledExtract()` or `LabeledExpand()`.
 | 0x0001 | AES-128-GCM      | 16  | 12  | {{GCM}}      |
 | 0x0002 | AES-256-GCM      | 32  | 12  | {{GCM}}      |
 | 0x0003 | ChaCha20Poly1305 | 32  | 12  | {{?RFC8439}} |
+| 0xFFFF | (reserved)       | N/A | N/A | N/A          |
```

Comment:
```suggestion
| 0xFFFF | Export-only      | N/A | N/A | [[RFCXXXX]] |
```

["Reserved" has a special meaning with IANA](https://tools.ietf.org/html/rfc8126#section-6).  It means that a value is neither assigned nor available for assignment.  This one should be assigned.